homeassistant/components/jvc_projector/translations/en.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
Please remove this translation file.
A recent change means translations have been fully externalised.

Reply:
Done!